curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -959,9 +962,9 @@ void MetaServerClientImpl::UpdateInodeAttrAsync(
 
 void MetaServerClientImpl::UpdateInodeAttrWithOutNlinkAsync(
```

Comment:
UpdateInodeAttrWithOutNlinkAsync  not only update attr also s3info, should rename this function